tests/test_jenkins.py

```
@@ -1,10 +1,47 @@
 
 
+class TestJenkinsClientInit:
+    """Tests for JenkinsClient initialization."""
+
+    @patch("jenkins_job_insight.jenkins.jenkins.Jenkins.__init__", return_value=None)
+    def test_timeout_passed_to_parent(self, mock_init):
+        """Test that timeout parameter is passed to parent Jenkins class."""
+        JenkinsClient(
+            url="http://jenkins.example.com",
+            username="user",
+            password="pass",  # pragma: allowlist secret
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to argument: "password"

(S106)